core/src/main/java/com/google/errorprone/refaster/Inliner.java

```diff
@@ -224,12 +224,9 @@ public TypeVar inlineAsVar(UTypeVar var) throws CouldNotResolveImportException {
     }
     Name name = asName(var.getName());
     TypeSymbol sym = new TypeVariableSymbol(0, name, null, symtab().noSymbol);
-    typeVar = new TypeVar(sym, /* bound= */ null, /* lower= */ symtab().botType);
+    typeVar = new TypeVar(sym, var.getUpperBound().inline(this), var.getLowerBound().inline(this));
     sym.type = typeVar;
     typeVarCache.put(var.getName(), typeVar);
-    // Any recursive uses of var will point to the same TypeVar object generated above.
-    typeVar.bound = var.getUpperBound().inline(this);
```

Comment:
These fields are removed and are instead put in the constructor. Looking at the code, it seems to make sense to put them in the constructor in the first place, since we would still override them. This should be 8/11-compatible.